docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-115.adoc

```diff
@@ -0,0 +1,49 @@
+== Azure Machine Learning web service API key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_115
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+An Azure Machine Learning web service API key is essential for accessing and utilizing Azure Machine Learning web services. This key is critical for authenticating requests to the service and ensuring that only authorized applications can interact with it. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data breaches. Securely managing this key is crucial to maintaining the security and integrity of your applications. Regularly reviewing and updating your security settings and keys is important to prevent unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://learn.microsoft.com/en-us/purview/sit-defn-azure-machine-learning-web-service-api-key[Azure Machine Learning documentation and support] resources.
+
+=== Fix - Buildtime
+
+*Azure*
+
+If your Azure Machine Learning web service API key is exposed, it is crucial to take immediate action to revoke the compromised key and generate a new one to secure access to the service. Follow these steps to replace the exposed key and update all dependent applications:
+
+1. Log into your Azure Portal using your administrator credentials.
+
+2. Navigate to the relevant Azure Machine Learning workspace.
+
+3. Access the 'Keys' section within the Azure Machine Learning service.
+
+4. Identify the exposed key and delete it.
+
+5. Generate a new API key by selecting 'Create new key'. Provide a name for the new key and save the changes.
```

Comment:
Select 'Create new key' to generate a new API Key. Provide a name for the new key and save the changes.